docs/ldml/tr35.md

```diff
@@ -3575,23 +3575,26 @@ The data from supplementalMetadata is (logically) preprocessed as follows.
 4. Change the _type_ and _replacement_ values in the remaining rules into multimap rules, as per _Definition 1. Multimap Interpretation_.
    1. Note that the “und” value disappears.
 5. Order the set of rules by
-   1. the size of the union of all field value sets, with largest size first
-   2. and then alphabetically by field.
+   1. The size of the union of all field value sets, with largest size first
+   2. And then by field, where L < S < R < V. 
+     * So {L=X, S=Z} < {L=X, R=Y}.
+     * After this point we are guaranteed to have the same set of fields
+   3. And then by field values sets in the order L, S, R, V. 
+     * To determine the ordering between a field set A and B, traverse each in parallel
+     * If the field values are all equal, the result is equal.
+     * Otherwise at the first differing field values, A is less if its field value is less, otherwise B is less.
```

Comment:
The first differing value is fonipa vs hepburn, so the fonipa rule comes first. That's what "Otherwise at the first differing field values, A is less if its field value is less, otherwise B is less." covers.

Reply:
The data in the file is in the form of a _ separated string , such as 
<languageAlias type="und_hepburn_heploc" replacement="und_alalc97" reason="deprecated"/>
and we change und_hepburn_heploc to the V={hepburn, heploc} notation here. If data is somehow
<languageAlias type="und_heploc_hepburn" replacement="und_alalc97" reason="deprecated"/>
should it still be V={hepburn, heploc}? which mean we need sort the fields in V ? 
or we should just in the data check in process ensure that will never happen?
My understand the {A, B} is a set notation not an ordered list notation (please correct me if I misunderstand) and if that is case then "the first" in a set does not make sense unless we specifcy the order in the set.